src/NATS.Client.Core/NATS.Client.Core.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
```

Comment:
Is there any point in targeting both right now?  Since we want to support the .NET LTS Release, and `net6.0` is supported until November 12, 2024, should we just stay on `net6.0` now.  Then on November 12, 2024 move the target to `net8.0`.

If there was a compelling enough use case for a `net8.0`-only API we could target both at that time, and use preprocessor directives to support both `net6.0` and `net8.0` until then.

Reply:
See the changes in JetStream models.

Reply:
Ah.  But shouldn't only that one assembly need the `net.8.0` target instead of all of them?

Reply:
I think it's good to have it on across the board since it would give us warnings if there is an obsolete apis or suggestions. There is one about the cancellation token source new `CancelAsync()` method for example.

Reply:
Another example is it makes it easier to spot trimming warnings.

Reply:
OK, let's document a policy to tag every assembly with the supported .NET LTS Releases then, and move forward with this